.github/workflows/e2e-component-pipeline.yml

```diff
@@ -0,0 +1,164 @@
+---
+name: Component-Pipeline
```

Comment:
I think we have a pipeline with this name. Can we rename it to something like `Scheduled-Component-Pipeline` and rename the file similarly.
Please check this for both pipelines.